main.go

```diff
@@ -107,7 +122,7 @@ func execute(cmd *cobra.Command, args []string) error {
 	}
 
 	// create an io.Reader from the markdown source in cli-args
-	src, err := readerFromArg(arg)
+	src, err := readerFromArg(cmd, arg)
```

Comment:
We probably need to split this up a bit. I'm not a fan of dragging the entire cmd into the little helper function. It also leads to other weird situations, where `-w` would be currently ignored if it's started with "github.com/foo/bar" as the source.